nengo_loihi/builder.py

```diff
@@ -326,10 +326,11 @@ def build_lif(model, lif, neurons, group):
         dt=model.dt)
 
 
-# TODO: fix issues with ReLu before enabling it
-# @Builder.register(nengo.RectifiedLinear)
+@Builder.register(nengo.RectifiedLinear)
 def build_relu(model, relu, neurons, group):
-    group.configure_relu(dt=model.dt)
+    group.configure_relu(
+        vth=1./model.dt,  # so input == 1 -> neuron fires 1/dt steps -> 1 Hz
```

Comment:
Are there any advantages/disadvantages to scaling the threshold versus scaling the input current?  I'm thinking of the scaling we do during discretization in particular, I know some of that is based on Vth.  Will changing Vth here affect the accuracy of the discretization or anything like that?

Reply:
I think the discretization should end up the same either way. The reason I did it on the threshold is because that was easy to do in the neuron builder, whereas scaling the input would mean checking for RectifiedLinear in the ensemble builder and then scaling gain and bias (a bit messier).